openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/OpenEdgeComponents.java

```diff
@@ -122,10 +115,10 @@ private void registerLicences(LicenceRegistrar[] licRegistrars, String permanent
       LOG.debug("Found {} licence - Permanent ID '{}' - Customer '{}' - Repository '{}' - Expiration date {}",
           lic.getType().toString(), lic.getPermanentId(), lic.getCustomerName(), lic.getRepositoryName(),
           DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.LONG).format(new Date(lic.getExpirationDate())));
-      if (!lic.getPermanentId().isEmpty() && !permanentId.equals(lic.getPermanentId())) {
-        LOG.debug("Skipped licence as it doesn't match permanent ID '{}'", permanentId);
-        continue;
-      }
+//      if (!lic.getPermanentId().isEmpty() && !permanentId.equals(lic.getPermanentId())) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)